app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -183,49 +179,71 @@ fun ViewProfileScreen(
                   colorLevel = ColorLevel.TERTIARY,
                   testTag = "EditButton")
 
-              // Scrollable Column instead of LazyColumn
-              Column(
-                  modifier =
-                      Modifier.fillMaxWidth()
-                          .verticalScroll(rememberScrollState())
-                          .padding(top = 56.dp) // Adjust based on button height
-                          .testTag("ProfileContentSections")) {
-                    FavoriteParkingsSection(userViewModel, parkingViewModel, navigationActions)
-
-                    Spacer(modifier = Modifier.height(16.dp))
-
-                    UserReviewsSection(reviewViewModel, userViewModel, parkingViewModel)
-                  }
+              // Display the TabLayout
+              TabLayout(
+                  userViewModel = userViewModel,
+                  parkingViewModel = parkingViewModel,
+                  reviewViewModel = reviewViewModel,
+                  navigationActions = navigationActions)
             }
           }
         }
       }
 }
 
+/** Tab layout that displays different sections based on the selected tab. */
 @Composable
-private fun FavoriteParkingsSection(
+private fun TabLayout(
     userViewModel: UserViewModel,
     parkingViewModel: ParkingViewModel,
+    reviewViewModel: ReviewViewModel,
     navigationActions: NavigationActions
 ) {
-  val favoriteParkings = userViewModel.favoriteParkings.collectAsState().value
+  var selectedTabIndex by rememberSaveable { mutableIntStateOf(0) }
+  val tabs =
+      listOf(
+          stringResource(R.string.view_profile_screen_favorite_parkings),
+          stringResource(R.string.view_profile_screen_my_reviews))
+
+  Column(
+      modifier = Modifier.fillMaxWidth().padding(top = 56.dp) // Adjust based on button height
+      ) {
+        TabRow(selectedTabIndex = selectedTabIndex, modifier = Modifier.testTag("TabRow")) {
+          tabs.forEachIndexed { index, title ->
+            Tab(
+                text = { Text(title) },
+                selected = selectedTabIndex == index,
+                onClick = { selectedTabIndex = index },
+                modifier = Modifier.testTag("Tab${title.replace(" ", "")}"))
+          }
+        }
 
-  Text(
-      text = stringResource(R.string.view_profile_screen_favorite_parking_title),
-      style = MaterialTheme.typography.titleLarge,
-      modifier = Modifier.testTag("FavoriteParkingsTitle"))
+        when (selectedTabIndex) {
```

Comment:
making it like this will make it much easier to incorporate new tabs later, well done!